### PR TITLE
🎣 Add @webhook as shortcut to .extra of last webhook result

### PR DESCRIPTION
**Pull request by rowanseymour**
Addresses #663

### CODE REVIEW

File: flows/runs/run.go [hunk 1]
```diff
@@ -214,6 +215,18 @@ func (r *flowRun) RootContext(env utils.Environment) map[string]types.XValue {
 	}
 }
 
+func (r *flowRun) lastWebhookResponse() types.XValue {
+	for e := len(r.events) - 1; e >= 0; e-- {
```
Comment (nicpottier) [inline flows/runs/run.go]: minor obviously but `e` is an odd choice for index no?

Comment (rowanseymour) [reply flows/runs/run.go]: Interesting, I guess go's convention of calling the `this` variable a single letter has taken me down a path of using the first letter of thing being looped, rather than the traditiona `i`, `j`, `k`,.. so for example:

```go
func (i *Input) foo() {
  s, session := range i.sessions {
     e, event := range s.events {
```

Comment (nicpottier) [reply flows/runs/run.go]: `i` in sacrosanct in my world, but ya maybe I'm wrong. Perhaps worth investigating what the stdlib does.

Comment (nicpottier) [reply flows/runs/run.go]: Perhaps just the strings library, but first place I looked seems to agree:
  https://golang.org/src/strings/strings.go?s=9816:9869#L373

Comment (ericnewcomer) [reply flows/runs/run.go]: Iterating a list of objects is one thing, but a counter? oof.

Comment (rowanseymour) [reply flows/runs/run.go]: I see some `n` and `p` but yeah `i` is most common. I don't mind sticking to `i`'s but if stdlib is the authority then you can't get upset when I do `if err := foo(); err != nil` which is all over the stdlib 😸

Comment (nicpottier) [reply flows/runs/run.go]: touche

Comment (nicpottier) [reply flows/runs/run.go]: I don't see any `n` or `p` loop vars though, where? Just out of curiosity.

Comment (nicpottier) [reply flows/runs/run.go]: And on the `if` construct, it isn't that I disagree that that is most common it is just that I am too dumb to parse it. It literally takes me a minute to figure out what is happening. I guess I'm just not smart enough for that cleverness is all, so I find the likelihood of me writing a bug that way much higher that if I do the simpler construct.

Comment (rowanseymour) [reply flows/runs/run.go]: Here's some `i`, `n`, `x` and `y` all together https://github.com/golang/go/blob/50bd1c4d4eb4fac8ddeb5f063c099daccfb71b26/src/runtime/memmove_test.go#L31

You can certainly make some ugly `if` statements but the exact same thing is true of `for` loops. I'm not sure `if err := foo(); err != nil` really is so much harder to parse than `for i := foo(); i < something;`

Comment (nicpottier) [reply flows/runs/run.go]: Those are while loops, but noted.

Hey, not arguing the construct, maybe I'll get used to it, just saying for me personally as much as I would love to be able to parse them no problem, I just can't. It isn't a style thing for me at all, its a not smart enough thing.

Comment (nicpottier) [reply flows/runs/run.go]: Apparently I can't tell what a while loop is either, nm that previous comment :)

Comment (rowanseymour) [reply flows/runs/run.go]: I don't think anyone buys you not being smart enough but anyways. We should write a style guide one day. 

One thing I'm noticing just flicking thru stdlib is they really love their single char var names.. not just for counters etc

Comment (nicpottier) [reply flows/runs/run.go]: I don't know what to say, I'm being 100% honest there, I just find it super hard to parse that construct. Given how much code we read, I just think it's best to choose the easier to read construct, if others find it easy to read and obvious all the more reason. Like I said, maybe I'd get used to it with time.

Comment (rowanseymour) [reply flows/runs/run.go]: Yeah but that has nothing to do with "smartness" so I'm not sure how helpful it is to keep claming that. It's simple that you don't use that construct so you don't recognize it as easily.

Comment (nicpottier) [reply flows/runs/run.go]: Perhaps what I'm trying to say is that if I don't find it super easy then maybe others won't either. Totally willing to acknowledge that may change with use. To me it just isn't worth the downsides to save one line.

Super happy to have the team discuss and be outvoted.

Did you merge without changing the `i`? I thought we came to agreement there.

Comment (rowanseymour) [reply flows/runs/run.go]: I was going to fix the loop variables afterwards. That style of using the first letter of the thing being enumerated is everywhere in goflow.

Comment (nicpottier) [reply flows/runs/run.go]: Ah ok.

Comment (rowanseymour) [reply flows/runs/run.go]: Btw looks like for maps `k` (key) and `v` (value) is the norm, e.g. https://github.com/golang/go/blob/master/src/archive/tar/common.go#L457
